source/c.txt

```diff
@@ -75,6 +76,37 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<http://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB server version 5.0 and C Driver version 1.18. When
+you use the Versioned API feature with an official MongoDB driver, you
+can update your driver or server without worrying about backward
+compatibility issues for any commands covered by the Versioned API. To
+use the Versioned API, use the :atlas:`Atlas connection string
```

Comment:
I think the primary requirement is specifying the version of the Versioned API, so we want to mention that first rather than mention that it depends on having a connection string. All your MongoDB commands also depend on having a connection string that establishes a connection to a cluster/instance, but we probably don't need to mention that.

Reply:
suggestion accepted-- dropping reference to the connection string altogether as it is clear within the code snippet and more info available elsewhere on the page.